File: LLM-Translations/GPT-4/codenet/Java/C/s575985145.c
#include <stdio.h>

int main() {
    int size, id, cols, i, j;
    scanf("%d", &size);
    int graph[size][size];

    for (i = 0; i < size; i++) {
        scanf("%d %d", &id, &cols);
        int row[size];
        for (j = 0; j < size; j++)
            row[j] = 0;
        for (j = 0; j < cols; j++) {
            int temp;
            scanf("%d", &temp);
            row[temp - 1] = 1;
        }
        for (j = 0; j < size; j++)
            graph[id - 1][j] = row[j];
    }

    for (i = 0; i < size; i++) {
        for (j = 0; j < size; j++) {
            printf("%d", graph[i][j]);
            if (j < size - 1)
                printf(" ");
        }
        printf("\n");
    }

    return 0;
}

// 

File: LLM-Translations/GPT-4/codenet/Java/Python/s575985145.py
import sys

def main():
    size = int(input())
    graph = [[0] * size for _ in range(size)]

    for _ in range(size):
        id_and_cols = list(map(int, input().split()))
        id = id_and_cols[0]
        cols = id_and_cols[1]
        row = [0] * size
        for j in range(2, len(id_and_cols)):
            row[id_and_cols[j] - 1] = 1
        graph[id - 1] = row

    for i in range(len(graph)):
        for j in range(len(graph[i])):
            print(graph[i][j], end=" ")
        print()

if __name__ == "__main__":
    main()


